experimental/packages/opentelemetry-instrumentation-xml-http-request/package.json

```diff
@@ -92,5 +92,6 @@
     "@opentelemetry/sdk-trace-web": "1.7.0",
     "@opentelemetry/semantic-conventions": "1.7.0"
   },
-  "homepage": "https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-instrumentation-xml-http-request"
+  "homepage": "https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-instrumentation-xml-http-request",
+  "sideEffects": false
```

Comment:
I believe xhr instrumentation is patching globals so this should be true ?

Reply:
Simply loading this instrumentation module doesn't patch the globals. As Webpack documented:

> A "side effect" is defined as code that performs a special behavior when imported, other than exposing one or more exports. An example of this are polyfills, which affect the global scope and usually do not provide an export.
> https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free

I believe this module is still side-effects free.

Reply:
LGTM